modules/nitrokey-blobs

```diff
@@ -1,11 +1,11 @@
 modules-$(CONFIG_NITROKEY_BLOBS) += nitrokey-blobs
 
 nitrokey-blobs_base_dir := nitrokey-blobs
-nitrokey-blobs_version := b629bad31046ff2f5f363656326646fe2081d033
+nitrokey-blobs_version := cba08e83d8bbd7d3470769afd7dbc8e61d6cd8b5
```

Comment:
This is referred by https://docs.dasharo.com/variants/novacustom_nv4x_adl/heads/#v090-2024-02-29

-> https://github.com/Dasharo/coreboot/tree/3a9aa3a4

You should review other blobs references here since Nitrokey maintains their ow coreboot blobs repo.